test/tests/functional/pbs_multi_sched.py

```diff
@@ -751,14 +751,12 @@ def test_pbsfs_revert_to_defaults(self):
         self.setup_sc1()
         a = {'queue_type': 'execution',
              'started': 'True',
-             'enabled': 'True'}
-        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq1')
-
-        p1 = {'partition': 'P1'}
-        self.server.manager(MGR_CMD_SET, QUEUE, p1, id='wq1', expect=True)
+             'enabled': 'True',
+             'partition': 'P1'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq1', expect=True)
         a = {'resources_available.ncpus': 2}
-        self.server.create_vnodes('vnode', a, 1, self.mom)
-        self.server.manager(MGR_CMD_SET, NODE, p1, id='vnode[0]', expect=True)
+        self.server.manager(MGR_CMD_SET, NODE, {'partition': 'P1'},
+                            id=self.mom.shortname, expect=True)
```

Comment:
Shouldn't this be partition and resources_available.ncpus?

Reply:
Oops! I missed it.  Changed it now. Please review.
[test_pbsfs_revert_to_defaults.log](https://github.com/PBSPro/pbspro/files/1776656/test_pbsfs_revert_to_defaults.log)